bindings/wasm/examples/worker.test.js

```diff
@@ -58,29 +50,16 @@ async function runExample(name) {
         URL.revokeObjectURL(glbURL);
         glbURL = e.data.glbURL;
         if (glbURL == null) {
-          reject('no glbURL)');
-        }
-        const docIn = await io.read(glbURL);
-        const nodes = docIn.getRoot().listNodes();
-        for (const node of nodes) {
-          const docMesh = node.getMesh();
-          if (!docMesh) {
-            continue;
-          }
-          const {mesh} = readMesh(docMesh);
```

Comment:
So, the one problem I see with this refactor is we're no longer testing `readMesh`, and we're not really testing `writeMesh` either since the test isn't validating its output at all. These functions aren't actually dependent on `Manifold`, so perhaps we can slot them back in?

Reply:
Actually we do test writeMesh, in the `worker.ts` code I am reading the output of `writeMesh` for the property.

Reply:
Ah, I see it now - okay, that's perfect. Thanks!